pkg/client/create_builder.go

```diff
@@ -323,6 +361,44 @@ func (c *Client) addConfig(ctx context.Context, kind string, config pubbldr.Modu
 	return nil
 }
 
+func (c *Client) processBuilderCreateTargets(ctx context.Context, opts CreateBuilderOptions) ([]dist.Target, error) {
+	var targets []dist.Target
+
+	// For backward compatibility, we must create a target with de default platform used in the Fetch implementation for daemon and remote
```

Comment:
```suggestion
	// For backwards compatibility, we must create a target with the default platform used by the image fetcher (daemon or remote)
```

Reply:
Why is this? It's a little confusing that we MUST fill in a target. Does the fetcher no longer have a default?